streaming/rrdpush.c

```diff
@@ -364,6 +364,21 @@ void rrdpush_send_labels(RRDHOST *host) {
 
     host->labels_flag &= ~LABEL_FLAG_UPDATE_STREAM;
 }
+
+void rrdpush_claimed_id(RRDHOST *host)
+{
+    if(unlikely(!host->rrdpush_send_enabled || !host->rrdpush_sender_connected))
+        return;
+
+    sender_start(host->sender);
+    netdata_mutex_lock(&host->claimed_id_lock);
+
+    buffer_sprintf(host->sender->build, "CLAIMED_ID %s %s\n", host->machine_guid, (host->claimed_id ? host->claimed_id : "NULL") );
+
+    netdata_mutex_unlock(&host->claimed_id_lock);
+    sender_commit(host->sender);
```

Comment:
After writing into the sender buffer the sender thread should be signalled over the pipe so that it knows there is data waiting.

Reply:
There is an example on https://github.com/netdata/netdata/blob/9ecb4756d7104a2dd11e74ecec31f413035c69bb/streaming/rrdpush.c#L362

Reply:
added